contracts/Ownable.sol

```diff
@@ -0,0 +1,130 @@
+// SPDX-License-Identifier: GPL-3.0
+
+pragma solidity ^0.8.0;
+
+import "./external/openzeppelin/Context.sol";
+
+/**
+ * @title Abstract ownable contract that can be inherited by other contracts
+ * @notice Contract module which provides a basic access control mechanism, where
+ * there is an account (an owner) that can be granted exclusive access to
+ * specific functions.
+ *
+ * By default, the owner is the deployer of the contract.
+ *
+ * The owner account is set through a two steps process.
+ *      1. The current `owner` calls {transferOwnership} to set a `pendingOwner`
+ *      2. The `pendingOwner` calls {acceptOwnership} to accept the ownership transfer
+ *
+ * The manager account needs to be set using {setManager}.
+ *
+ * This module is used through inheritance. It will make available the modifier
+ * `onlyOwner`, which can be applied to your functions to restrict their use to
+ * the owner.
+ */
+abstract contract Ownable is Context {
+    address private _owner;
+    address private _pendingOwner;
+
+    /**
+     * @dev Emitted when `_pendingOwner` has been changed.
+     * @param pendingOwner new `_pendingOwner` address.
+     */
+    event OwnershipOffered(address indexed pendingOwner);
+
+    /**
+     * @dev Emitted when `_owner` has been changed.
+     * @param previousOwner previous `_owner` address.
+     * @param newOwner new `_owner` address.
+     */
+    event OwnershipTransferred(address indexed previousOwner, address indexed newOwner);
+
+    /* ============ Deploy ============ */
+
+    /**
+     * @notice Initializes the contract setting the deployer as the initial owner.
+     */
+    constructor() {
```

Comment:
Pass the owner to the constructor, otherwise deploying will be hard

Reply:
Done: https://github.com/pooltogether/owner-manager-contracts/pull/2/commits/c29bd611dc008202d7ecb499d26972256bc00179